sklearn/gaussian_process/_gpr.py

```diff
@@ -93,12 +93,11 @@ def optimizer(obj_func, initial_theta, bounds):
         run is performed.
 
     normalize_y : boolean, optional (default: False)
-        Whether the target values y are normalized, i.e., the mean of the
-        observed target values become zero. This parameter should be set to
-        True if the target values' mean is expected to differ considerable from
-        zero. When enabled, the normalization effectively modifies the GP's
```

Comment:
Why is this comment about the theoretical concerns of using this parameter removed?

Reply:
Now this is a tricky one! Partly because discussions around the likelihood principle can go on for ever... 

Really my concern here is a practical one. The comment seems to be telling the user that they should not normalise the data but, as we now know, the model only actually provides predictions whose standard deviation can only be between 0 and 1. So, normalisation is essential in this case, given the model's limitations. I think it's important that we don't put the user off from using the normalisation as, for this implementation, it's really crucial.